system/HTTP/Files/UploadedFile.php

```diff
@@ -141,7 +141,7 @@ public function move(string $targetPath, ?string $name = null, bool $overwrite =
         $destination = $overwrite ? $targetPath . $name : $this->getDestination($targetPath . $name);
 
         try {
-            move_uploaded_file($this->path, $destination);
+            $this->hasMoved = move_uploaded_file($this->path, $destination);
```

Comment:
When `move_uploaded_file()` returns `false`, the current code returns `false` after all.
But we need to throw `HTTPException::forMoveFailed()` as the same as `move_uploaded_file()` raise warning
to have consistent output in case of failure.

Please change the code like that.